src/Lykke.Service.QuotesHistory/Startup.cs

```diff
@@ -43,30 +44,53 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 services.AddMvc()
                     .AddJsonOptions(options =>
                     {
+                        options.SerializerSettings.Converters.Add(new StringEnumConverter());
                         options.SerializerSettings.ContractResolver =
                             new Newtonsoft.Json.Serialization.DefaultContractResolver();
                     });
 
                 services.AddSwaggerGen(options =>
                 {
                     options.DefaultLykkeConfiguration("v1", "Lykke.Service.QuotesHistory API");
+
+                    //Determine base path for the application.
+                    var basePath = PlatformServices.Default.Application.ApplicationBasePath;
+
+                    //Set the comments path for the swagger json and ui.
+                    var xmlPath = Path.Combine(basePath, "Lykke.Service.QuotesHistory.xml");
```

Comment:
This is already solved in the Common.APiLibrary as far as I remember. Consider to use it